src/main/resources/com/rsmart/kuali/kfs/fp/spring-fp.xml

```diff
@@ -100,7 +100,7 @@
 		<property name="vendorService" ref="vendorService"/>
 		<property name="batchFeedHelperService" ref="batchFeedHelperService"/>
 		<property name="attachmentsPath">
-	       <value>${staging.directory}/fp/disbursementVoucher</value>
+	       <value>${staging.directory}/fp/disbursementVoucher/attachment</value>
```

Comment:
That would make sense. I will update it. I was trying to be consistent with the existing vendor batch job that reads the attachments from: staging/vnd/vendorBatch/attachment forlder. See VendorBatchServiceImpl line 551.